lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -348,7 +333,7 @@ type JoinStakePool n = "stake-pools"
     :> "wallets"
     :> Capture "walletId" (ApiT WalletId)
     :> ReqBody '[JSON] ApiWalletPassphrase
-    :> PutAcccepted '[JSON] (ApiTransaction n)
```

Comment:
While updating this, I also realized why I couldn't import `PutAccepted` in the past despite being present in the doc....

`PutAcccepted` vs `PutAccepted` ... subtle.. :man_facepalming: